File: Python/Problem #1 [Easy]/main.py
n_list = [10, 15, 3, 7]
k = 17

def solve(n_list, k): 
    for n in n_list:
        aux = k
        aux -= n

        if aux in n_list:
            print(f"{n} + {aux} = {k}")
            return True

    return False

solve(n_list, k)

File: TypeScript/Problem #1 [Easy]/main.ts
let n_list: Array<number> = [10, 15, 3, 7];
let k: number = 17;

let solve = (n_list, k) => {
    for(let n of n_list) {
        let aux: number = k;
        aux -= n;

        if (n_list.includes(aux)) {
            console.log(`${n} + ${aux} = ${k}`);
            return true;
        }
    }
    return false;
}

solve(n_list, k);